src/bindings/c/ov/tests/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME "OpenVinoCAPITests")
```

Comment:
Can we use snake_case?

Reply:
sure! OV API 2.0 adopts snake_case coding style.